product_search_by_display_name/models/product_product.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2023 Komit - Cuong Nguyen Mtm
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ProductProduct(models.Model):
+    _inherit = "product.product"
+
+    display_name = fields.Char(
+        automatic=True,
```

Comment:
i think automatic=True is already applied to all display_name fields

Reply:
This is somehow required otherwise the field lost the default display_name feature